ros2param/ros2param/verb/dump.py

```diff
@@ -97,6 +94,7 @@ def main(self, *, args):  # noqa: D102
             # create client
             service_name = f'{absolute_node_name}/list_parameters'
             client = node.create_client(ListParameters, service_name)
+            # TODO(ihasdapie): use api call
```

Comment:
I think this TODO still needs to be resolved.

Looks like we're calling the list parameters service twice now.